src/config/session-management.md

```diff
@@ -0,0 +1,38 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, but it can be used
+to safely create temporary runtime directories, provide access to hardware
+devices and multi-seat capabilities, and control system shutdown.
+
+## D-Bus
+
+D-Bus is an IPC (inter-process communication) mechanism used by userspace
+software in Linux. There are two uses for it: as a system bus, and as a user
+bus.
```

Comment:
i wonder whether we should say "user bus" or "session bus". The latter is the technical name for it (cf. e.g. the `DBUS_SESSION_BUS_*` env vars), and it might be better for readers if our terminology matches the terminology used elsewhere.

Reply:
I used session bus, but also made it clear that it's for a user session. Do you think it's a good explanation?

Reply:
Thanks, that looks good.